### PR TITLE
OWNERS: move awesomenix and justaugustus to emeritus

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**What type of PR is this?**
/kind other

**What this PR does / why we need it**: Moves @awesomenix and @justaugustus to emeritus maintainers as they haven't been active in the project for a while. Spoke to both of them before opening this PR.

Thank you @awesomenix and @justaugustus for all your contributions to CAPZ and for the huge part you both played in getting this provider where it's at today 👏  🙏 

Also updated CAPI maintainers to match the latest https://github.com/kubernetes-sigs/cluster-api/blob/master/OWNERS_ALIASES.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -7,21 +7,23 @@ aliases:
     - neolit123
     - timothysc
   cluster-api-admins:
-    - davidewatson
-    - detiber
     - justinsb
+    - detiber
     - vincepri
```
Comment (justaugustus) [inline OWNERS_ALIASES:12]:
```suggestion
    - detiber
    - justinsb
    - vincepri
```

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -7,21 +7,23 @@ aliases:
...
   cluster-api-maintainers:
-    - detiber
     - justinsb
+    - detiber
     - vincepri
+    - CecileRobertMichon
```
Comment (justaugustus) [inline OWNERS_ALIASES:17]:
```suggestion
    - CecileRobertMichon
    - detiber
    - justinsb
    - vincepri
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@CecileRobertMichon -- Alpha-sort nits, but outside of that, lgtm!

@kubernetes-sigs/cluster-api-provider-azure-maintainers -- It has an absolute pleasure to be a part of what was created here and to see so many rally around capz and take it further that I could've ever by myself.

Farewell (for now) and thank you for everything! 💕 

My final `/approve` (for now):
/approve

File: OWNERS_ALIASES (hunk 1 above)
Comment (CecileRobertMichon) [reply OWNERS_ALIASES:12]: ah I was lazy and copy pasted from CAPI's file... will fix

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -7,21 +7,23 @@ aliases:
...
-    - justaugustus
     - nader-ziada
   cluster-api-azure-reviewers:
     - alexeldeib
     - cpanato
     - juan-lee
+  emeritus_maintainers:
+    - awesomenix
+    - justaugustus
+    - ncdc
```
Comment (justaugustus) [inline OWNERS_ALIASES:29]:
```suggestion
```

File: OWNERS [hunk 4]
```diff
@@ -10,9 +10,6 @@ reviewers:
   - cluster-api-azure-maintainers
   - cluster-api-azure-reviewers
```
Comment (justaugustus) [inline OWNERS:12]:
```suggestion

  emeritus_approvers:
    - awesomenix
    - justaugustus
    - ncdc
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@CecileRobertMichon -- sorry, one more nit on re-reviewing the [emeritus section](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#emeritus) of the `OWNERS` docs.

The emeritus key should be in the `OWNERS` file and the correct key is `emeritus_approvers`.
Left a suggestion! 🙃

**Comment by justaugustus** [comment]
Hehe, the bots don't want us to leave!
/retest
/lgtm

**Review by cpanato — APPROVED** [review]
thanks, Stephen, just thanks for all 💕 

/lgtm

**Comment by devigned** [comment]
/lgtm

Thank you @justaugustus and @awesomenix!

**Comment by nader-ziada** [comment]
/lgtm

Thank you for all your great work @justaugustus and @awesomenix

**Pull request merged**